client/ratings_reviews/RatingsReviews.jsx

```diff
@@ -46,6 +46,9 @@ class RatingsReviews extends React.Component {
     }).catch((error) => {
       console.log(error);
     });
+
+    sessionStorage.setItem('helpfulReviews', JSON.stringify([]));
```

Comment:
Curious why you chose sessionStorage over localStorage - I believe the data in localStorage doesn't expire (unless you manually clear it), but sessionStorage may expire when the session ends. Do you know if this still works if you close the browser window and re-open it at the URL? Also, do you have to store an empty array at this point, or could you store the helpfulReviews later, once you have at least one item to store there?

Reply:
Oh I think I see why! That way you can just push to the array later (in your ReviewTile component), and not have to worry about creating an array at that point. At least I think that's why? Haha

Reply:
HI @jabrake, thank you so much for pointing this out! I chose sessionStorage because I don't actually want the data to persist after the session ends. Everything that I store in there is just for the current user so that I don't need to make additional AJAX requests during a session to retrieve the more updated helpful/report status from the API over and over every time the user clicks a helpful/report button. This is the reason why I reset the sessionStorage anyways (just to be sure) to be an empty array. 

And yes, if I refresh the page, both helpful and report status should be updated automatically because AJAX requests are made to retrieve the most updated data from the API whenever the page first gets rendered.

Not sure if I approach this correctly, so any additional comments are very much appreciated! Thank you again!